explorer/app/apis/azul/anvil-cmg/common/transformers.ts

```diff
@@ -126,7 +128,7 @@ export function getDatasetDescription(
  * @returns data summaries key-value pairs of data summary label and corresponding value.
  */
 export function getDatasetDetails(
-  response: DatasetEntityResponse
+  response: DatasetsResponse
```

Comment:
I changed the type used here and in `buildDatasetDetails` (hopefully to the correct type?), but I'm not sure whether the documentation comments need to be updated to reflect this somehow

Reply:
@hunterckx - update the JSDOC if you think it needs it?